infrastructure/main.tf

```diff
@@ -10,5 +10,6 @@ module "pdf-service-api" {
     REFORM_TEAM = "${var.product}"
     REFORM_SERVICE_NAME = "${var.microservice}"
     REFORM_ENVIRONMENT = "${var.env}"
+    S2S_URL = "http://idam-s2s-${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
```

Comment:
does S2s exist on CNP?
Last I heard from Jason was the plan was that S2s would stay on tactical

Reply:
After talking to a few more people at PI and Brendt I think we will have to migrate actually ;(

Reply:
it probably makes sense to migrate it before adding this PR 🎉